tracer/src/Datadog.Trace/Debugger/Instrumentation/MethodDebuggerInvoker.cs

```diff
@@ -267,14 +267,21 @@ internal static bool TryGetLocalName(int index, string[] localNamesFromPdb, out
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static void LogException(Exception exception, ref MethodDebuggerState state)
```

Comment:
nit: rename to TryLogException

Reply:
Makes sense, thanks! I won't address this now because it would require changes to native code that are out of scope for this PR, but will discuss with @dudikeleti